src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_isLoginLongEnough_notOkay() {
+        User user = new User("cat", "1234567", 19);
+        assertThrows(FailedRegistrationException.class, () -> registrationService.register(user),
+                "Test failed! Your method should throw FailedRegistrationException"

```

Comment:
you don't need this additional `message` parameter in `assertThrows`

your test should be self-explanatory (name, code inside etc)

**Applicable for all test methods**